data/jobs.txt

```diff
@@ -2808,4 +2863,3 @@ mission "Marauder Hunted"
 		system
 			distance 5 5
 		fleet "Marauder fleet X"
-	
```

Comment:
Looks like your editor deleted the trailing blank line at the end of the file. Can you re-add it?

Reply:
Looks like there were previously two EoF newlines and it trimmed one, yeah.